core/vm/interpreter.go

```diff
@@ -174,21 +168,6 @@ func (in *EVMInterpreter) Run(contract *Contract, input []byte, readOnly bool) (
 		}()
 	}
 
-	// Evaluate one address per group of 256, 31-byte chunks
-	if in.evm.ChainConfig().IsCancun(in.evm.Context.BlockNumber) && !contract.IsDeployment {
-		contract.Chunks = trie.ChunkifyCode(contract.Code)
-
-		// number of extra stems to evaluate after the header stem
-		extraEvals := (len(contract.Chunks) + 127) / verkle.NodeWidth
-
-		chunkEvals = make([][]byte, extraEvals+1)
-		for i := 1; i < extraEvals+1; i++ {
-			chunkEvals[i] = utils.GetTreeKeyCodeChunkWithEvaluatedAddress(contract.AddressPoint(), uint256.NewInt(uint64(i)*256))
-		}
-		// Header account is already known, it's the header account
-		chunkEvals[0] = utils.GetTreeKeyVersionWithEvaluatedAddress(contract.AddressPoint())
-	}
```

Comment:
This is where we avoid the situation of having to calculate the `GetTreeKeyCodeChunkWithEvaluatedAddress` of all the chunks of a contract, when we might be only touching some in the contract execution (L175 loop)